website/content/en/preview/tasks/configuration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Configuration"
+linkTitle: "Configuration"
+weight: 5
+description: >
+  Configure Karpenter
+---
+
+Karpenter supplies a default configuration that should work for most.  Additional configuration can be performed by 
+editing the `config` configmap within the namespace that Karpenter was installed in.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+data:
+  # The maximum length of a batch window. The longer this is, the more pods we can consider for provisioning at one
+  # time which usually results in fewer but larger nodes.
+  batchIdleDuration: 1s
+  # The maximum amount of time with no new ending pods that if exceeded ends the current batching window. If pods arrive
+  # faster than this time, the batching window will be extended up to the maxDuration. If they arrive slower, the pods
+  # will be batched separately.
+  batchMaxDuration: 10s
+```
+
+## Batching Parameters
+
+The batching parameters control how Karpenter batches an incoming stream of pending pods.  Reducing these values may trade off a slightly faster time from pending pod to node launch, in exchange for launching smaller nodes.  Increasing the values can do the inverse.  Karpenter provides reasonable defaults for these values, but if you have specific knowledge about your workloads you can tweak these parameters to match the expected rate of incoming pods.
+
+For a standard deployment scale-up, the pods arrive at the QPS setting of the `kube-controller-manager`, and the default values are typically fine.  These settings are intended for use cases where other systems may create large numbers of pods over a period of many seconds or minutes and there is a desire to batch them together.
+
+### `batchIdleDuration`
+
+The `batchIdleDuration` is the period of time that a new pending pod extends the current batching window. This can be increased to handle scenarios where pods arrive slower than one second part, but it would be preferable if they were batched together onto a single larger node.   
+
+This value is expressed as a string value like "10s", "1m" or "2h45m". The valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

Comment:
I think backticks might be better than quotes in this case, as in `10s`.

Reply:
SGTM